apps/crn-server/test/controllers/research-outputs.test.ts

```diff
@@ -571,7 +568,34 @@ describe('ResearchOutputs controller', () => {
       ).rejects.toThrow(NotFoundError);
     });
 
-    test('Should update the research output and return it', async () => {
+    test('Should update a published existing research output and return it', async () => {
+      const researchOutputUpdateData: ResearchOutputUpdateData = {
+        ...getResearchOutputUpdateData(),
+      };
+      researchOutputDataProviderMock.update.mockResolvedValueOnce(
+        researchOutputId,
+      );
+
+      const result = await researchOutputs.update(
+        researchOutputId,
+        researchOutputUpdateData,
+        { publish: false },
+      );
+
+      const researchOutputUpdateDataObject =
+        getResearchOutputUpdateDataObject();
+
+      expect(result).toEqual(getResearchOutputResponse());
+      expect(researchOutputDataProviderMock.update).toBeCalledWith(
+        researchOutputId,
+        researchOutputUpdateDataObject,
+        { publish: false },
```

Comment:
to be fair i am in two minds about this being a separate parameter. Like I mentioned earlier in this review (and I think so did @kmaid) it feels like we could just infer the actual state of the research-output by simply looking at the `published` property rather than have this separate 'option'